projects/thrift/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Copyright 2021 Google LLC
+#
```

Comment:
https://www.apache.org/legal/src-headers.html

Reply:
@inferno-chromium could you help me with the legal side cf https://google.github.io/oss-fuzz/getting-started/new-project-guide/#copyright-headers ?